schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1401 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema.py
+
+Node: dcid:fanconi_Anaemia_Mutation_Database_(FANCM)_ID
+name: "fanconi_Anaemia_Mutation_Database_(FANCM)_ID"
```

Comment:
Can we make these property names camelCase ?

We use '_' in property or class names very sparringly.

Reply:
changed to camel case